app/assets/javascripts/discourse/tests/acceptance/user-menu-test.js

```diff
@@ -0,0 +1,96 @@
+import { click, visit } from "@ember/test-helpers";
+import {
+  acceptance,
+  query,
+  updateCurrentUser,
+} from "discourse/tests/helpers/qunit-helpers";
+import { test } from "qunit";
+import { cloneJSON } from "discourse-common/lib/object";
+import TopicFixtures from "discourse/tests/fixtures/topic";
+import I18n from "I18n";
+
+acceptance("User menu", function (needs) {
+  needs.user({ redesigned_user_menu_enabled: true });
+  let requestHeaders = {};
+  needs.pretender((server, helper) => {
+    server.get("/t/1234.json", (request) => {
+      const json = cloneJSON(TopicFixtures["/t/130.json"]);
+      json.id = 1234;
+      json.post_stream.posts.forEach((post) => {
+        post.topic_id = 1234;
+      });
+      requestHeaders = request.requestHeaders;
+      return helper.response(json);
+    });
+  });
```

Comment:
```suggestion
  needs.user({ redesigned_user_menu_enabled: true });
  let requestHeaders = {};
  
  needs.pretender((server, helper) => {
    server.get("/t/1234.json", (request) => {
      const json = cloneJSON(TopicFixtures["/t/130.json"]);
      json.id = 1234;
      json.post_stream.posts.forEach((post) => {
        post.topic_id = 1234;
      });
      requestHeaders = request.requestHeaders;
      return helper.response(json);
    });
  });
```

I know the lint doesn't cover it but I think leaving a blank line for multi-line code blocks helps to improve the readability of the code.